test/tests.json

```diff
@@ -11,6 +11,9 @@
     ["test@nominet.org.uk", "valid"],
     ["test@about.museum", "valid"],
     ["a@iana.org", "valid"],
+    ["êjness@iana.com", "valid"],
+    ["ñoñó1234@iana.com", "valid"],
```

Comment:
@skeggse even with unicode character support, this fails on the DNS lookup because of `iana.com` vs `iana.org`.